### PR TITLE
Add DFX TVL KPI options implementation

### PR DESCRIPTION
**Pull request by Reinis-FRP**

### CODE REVIEW

File: Implementations/dfx-tvl.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+DFX TVL KPI options track value locked in following pool contracts:
+* DFXv0.5 CADC on Ethereum: [0xa6c0cbcaebd93ad3c6c94412ec06aaa37870216d](https://etherscan.io/address/0xa6c0cbcaebd93ad3c6c94412ec06aaa37870216d)
+* DFXv0.5 EURS on Ethereum: [0x1a4Ffe0DCbDB4d551cfcA61A5626aFD190731347](https://etherscan.io/address/0x1a4Ffe0DCbDB4d551cfcA61A5626aFD190731347)
+* DFXv0.5 XSGD on Ethereum: [0x2baB29a12a9527a179Da88F422cDaaA223A90bD5](https://etherscan.io/address/0x2baB29a12a9527a179Da88F422cDaaA223A90bD5)
+* DFXv0.5 CADC on Polygon: [0x288Ab1b113C666Abb097BB2bA51B8f3759D7729e](https://polygonscan.com/address/0x288Ab1b113C666Abb097BB2bA51B8f3759D7729e)
+* DFXv0.5 EURS on Polygon: [0xB72d390E07F40D37D42dfCc43E954Ae7c738Ad44](https://polygonscan.com/address/0xB72d390E07F40D37D42dfCc43E954Ae7c738Ad44)
+* DFXv0.5 XSGD on Polygon: [0x8e3e9cB46E593Ec0CaF4a1Dcd6DF3A79a87b1fd7](https://polygonscan.com/address/0x8e3e9cB46E593Ec0CaF4a1Dcd6DF3A79a87b1fd7)
+
+The list above can be amended once new pool contracts are developed by DFX protocol.
```
Comment (smb2796) [inline Implementations/dfx-tvl.md:39]: Is there an external canonical list, or are we just going to update this implementation doc?

**Review by smb2796 — APPROVED** [review]

File: Implementations/dfx-tvl.md (hunk 1 above)
Comment (Reinis-FRP) [reply Implementations/dfx-tvl.md]: Added DFX contract repo and newly deployed NZDS

**Review by smb2796 — APPROVED** [review]

File: Implementations/dfx-tvl.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+DFX TVL KPI options track value locked in following pool contracts:
+* DFXv0.5 CADC on Ethereum: [0xa6c0cbcaebd93ad3c6c94412ec06aaa37870216d](https://etherscan.io/address/0xa6c0cbcaebd93ad3c6c94412ec06aaa37870216d)
+* DFXv0.5 EURS on Ethereum: [0x1a4Ffe0DCbDB4d551cfcA61A5626aFD190731347](https://etherscan.io/address/0x1a4Ffe0DCbDB4d551cfcA61A5626aFD190731347)
+* DFXv0.5 XSGD on Ethereum: [0x2baB29a12a9527a179Da88F422cDaaA223A90bD5](https://etherscan.io/address/0x2baB29a12a9527a179Da88F422cDaaA223A90bD5)
+* DFXv0.5 NZDS on Ethereum: [0xE9669516e09f5710023566458F329cCE6437AaaC](https://etherscan.io/address/0xE9669516e09f5710023566458F329cCE6437AaaC)
+* DFXv0.5 CADC on Polygon: [0x288Ab1b113C666Abb097BB2bA51B8f3759D7729e](https://polygonscan.com/address/0x288Ab1b113C666Abb097BB2bA51B8f3759D7729e)
+* DFXv0.5 EURS on Polygon: [0xB72d390E07F40D37D42dfCc43E954Ae7c738Ad44](https://polygonscan.com/address/0xB72d390E07F40D37D42dfCc43E954Ae7c738Ad44)
+* DFXv0.5 XSGD on Polygon: [0x8e3e9cB46E593Ec0CaF4a1Dcd6DF3A79a87b1fd7](https://polygonscan.com/address/0x8e3e9cB46E593Ec0CaF4a1Dcd6DF3A79a87b1fd7)
+
```
Comment (therealhexi) [inline Implementations/dfx-tvl.md:39]: We don't have a New Zealand Dollar option being tracked on Polygon? I couldn't find it on Polygonscan, but am wondering why it's not consistent with Ethereum network

**Review by therealhexi — APPROVED** [review]

File: Implementations/dfx-tvl.md (hunk 2 above)
Comment (Reinis-FRP) [reply Implementations/dfx-tvl.md:39]: The list is based on DFX project [contract repo](https://github.com/dfx-finance/contracts). NZDS is new pool and they have just launched it on the Ethereum network. We would need to update it here if Polygon pool is deployed later on.

**Pull request merged**